src/item_contents.cpp

```diff
@@ -507,7 +504,8 @@ std::pair<item_location, item_pocket *> item_contents::best_pocket( const item &
             for( item *contained : all_items_top( item_pocket::pocket_type::CONTAINER ) ) {
                 std::pair<item_location, item_pocket *> internal_pocket =
                     contained->contents.best_pocket( it, parent, true );
```

Comment:
```suggestion
                    contained->contents.best_pocket( it, parent );
```
Please ensure all places where `item_contents::best_pocket` is used are updated.

Reply:
I have updated that but somehow not committed.. Weird.
Thanks for the diligence!